aim/ext/transport/client.py

```diff
@@ -76,15 +77,20 @@ def _create_remote_channels(self):
 
         ssl_certfile = os.getenv(AIM_CLIENT_SSL_CERTIFICATES_FILE)
         msg_max_size = int(os.getenv(AIM_RT_MAX_MESSAGE_SIZE, AIM_RT_DEFAULT_MAX_MESSAGE_SIZE))
+        bearer_token = os.getenv(AIM_RT_BEARER_TOKEN)
         options = [
             ('grpc.max_send_message_length', msg_max_size),
             ('grpc.max_receive_message_length', msg_max_size)
         ]
 
         # open a channel with router
         if ssl_certfile:
```

Comment:
this check also needs to be changed, so if the `bearer_token` is provided then we should go with the case of secure channel as well